src/types/types.ts

```diff
@@ -16,6 +17,8 @@ export enum LanguageModeType {
   Target = 'target',
 }
 
+export type LanguageCode = (typeof languages)[number];
```

Comment:
Unless we're actually using `const languages`, we could do this to infer LanguageCode from semantics: `export type LanguageCode = Params['sourceLanguage'];`

Reply:
Nice!

Reply:
Done ✔️